examples/workflow.yaml

```diff
@@ -1,43 +1,50 @@
-apiVersion: nerdalize.com/v1alpha1
-kind: Workflow
-metadata:
-  name: test-workflow
-spec:
-  activeDeadlineSeconds: 3600
-  steps:
-    step-a:
-      jobTemplate:
-        metadata:
-          name: job1
-        spec:
-          parallelism: 1
-          template:
-            metadata:
-              name: pod1
-            spec:
-              restartPolicy: OnFailure
-              containers:
-                - image: ubuntu
-                  name: ubuntu1
-                  command:
-                    - /bin/sleep
-                    - 30
-    step-b:
-      dependencies:
-        - step-a
-      jobTemplate:
-        metadata:
-          name: job2
-        spec:
-          parallelism: 1
-          template:
-            metadata:
-              name: pod2
-            spec:
-              restartPolicy: OnFailure
-              containers:
-                - image: ubuntu
-                  name: ubuntu2
-                  command:
-                    - /bin/sleep
-                    - 30
+---
+  apiVersion: "nerdalize.com/v1alpha1"
+  kind: "Workflow"
+  metadata: 
+    name: "test-workflow"
```

Comment:
I converted the json to yaml with an online converter. This is convenient as the yaml should always be the same as the json. For every change we'd need to remove all the quotes which is a lot of work. As the result is the same I think we can keep the quotes.